src/main/scala/beam/agentsim/infrastructure/ZonalParkingManager.scala

```diff
@@ -119,7 +119,7 @@ object ZonalParkingManager extends LazyLogging {
     seed: Int,
     mnlParkingConfig: BeamConfig.Beam.Agentsim.Agents.Parking.MulitnomialLogit,
     beamConfig: BeamConfig,
-    includesHeader: Boolean = true
+//    includesHeader: Boolean = true
```

Comment:
Let's just remove it and let git history be used if needed.

Reply:
@JustinPihony removed